.circleci/config.yml

```diff
@@ -59,6 +60,29 @@ jobs:
           name: DevOps checkout
           command: |
             git clone $DEVOPS_REPO $DEVOPS_WORKDIR
+      - run:
+          name: Start dev server
+          command: |
+            cp $DEVOPS_WORKDIR/environments/cypress .env
+            echo SOCKET_PORT=3001 >> .env
+            yarn dev || true
+          background: true
+      - run:
+          name: Wait for dev server to be healthy
+          command: curl -4 --retry 30 --retry-connrefused --retry-delay 10 http://localhost:3000/graphql
```

Comment:
This `background: true` [config property](https://circleci.com/docs/2.0/configuration-reference/#background-commands) allows the server task to keep running while continuing to the next steps. The nice part about this is that the `jest` output isn't co-mingled with the PM2 output, which I found to be more usable.